modules/ROOT/pages/pce-configuration.adoc

```diff
@@ -0,0 +1,17 @@
+= Configure Private Cloud Edition in Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
```

Comment:
INSTRUCTIONS for edit:

The modules/ROOT/pages/pce-configuration.adoc are instructions to point Studio to PCE instead of the  default US cloud platform. It would be useful if you could have it by EOD tomorrow. If not, Thursday works too,